### DISCUSSION TITLE
I want to use the version of Firmwares as sensors

### DISCUSSION THREAD

**Discussion by abuesen** [nathanmarlor/foxess_modbus#504] [Ideas]
It would be nice to get the versions of firmwares of the system as sensors in Homassistant.

The updates on my system are done by foxess or my electrician. So I do not know if a new firmware arrives. With the sensor it is possible to show and alert on change.

**Comment by canton7** [comment]
I've been thinking about this. It would be easy enough to do trivially, but that means polling the version every few seconds, which seems massively redundent. Instead, it would be better to poll once on (re-)connection. But, that means adding some logic which we don't currently have.

**Reply by canton7** [reply]
Opened #509
